images/win/scripts/Installers/Install-SQLExpress.ps1

```diff
@@ -0,0 +1,54 @@
+#Download web-installer
+Set-Item .\MaxMemoryPerShellMB 4096
+$installerUrl = "https://go.microsoft.com/fwlink/?linkid=866658"
+$installerName = "SQL2019-SSEI-Expr.exe"
+$installerPath = "${env:Temp}\$installerName"
+Write-Host "Download web-installer"
+(New-Object System.Net.WebClient).DownloadFile($installerUrl, $installerPath)
+#use web-installer to download setup package
+$downloadPath = "C:\SQLEXPRESS2019"
+$setupPath = Join-Path $downloadPath "SQLEXPR_x64_ENU"
+$downloadArgs = ("/MEDIAPATH=$downloadPath", "/MEDIATYPE=Core","/Action=Download", "/QUIET")
+$installArgs = ("/Q", "/IACCEPTSQLSERVERLICENSETERMS")
+$args = ("/c", "$setupPath/SETUP.exe", "/Q", "/IACCEPTSQLSERVERLICENSETERMS", "/Action=Install", "/INSTANCEID=SQL2019", "/INSTANCENAME=SQL2019", "/SECURITYMODE=SQL", "/SAPWD=P@ssword!!", "/TCPENABLED=1")
+Write-Host "Downloading SQL Express setup package"
```

Comment:
My concern is to create a separate function for every stage:
1. Downloading SQL
2. Unpack packages
3. Install SQL
4  etc.

Reply:
I have create two separate functions:
The first one for Start-Process tasks handling.
The second one for downloading process.

It should be enough to close this task I think. Please review this.